x-pack/plugins/maps/server/maps_telemetry/util.ts

```diff
@@ -73,6 +73,16 @@ export interface TELEMETRY_TERM_JOIN_COUNTS_PER_CLUSTER {
   [TELEMETRY_TERM_JOIN]?: ClusterCountStats;
 }
 
+export enum TELEMETRY_GRID_RESOLUTION {
```

Comment:
The code below works equally well just using `GRID_RESOLUTION` (tested locally). I don't think we need to define a new enum here with lowercase values.

Reply:
Just kept it separate to remain consistent with the others. It will break extensibility of the `grid_resolution`-type if we don't, because the telemetry-schema will get caught up  in it.